packages/hdwallet-native/src/bitcoin.ts

```diff
@@ -193,13 +197,13 @@ export function MixinNativeBTCWallet<TBase extends core.Constructor<NativeHDWall
           case "p2sh-p2wpkh":
           case "p2sh":
           case "bech32":
-            scriptData.redeemScript = payment.redeem?.output;
+            scriptData.redeemScript = payment.redeem?.output ? Buffer.from(payment.redeem.output) : undefined;
             break;
         }
 
         const bchData: BchInputData = {};
         if (coin.toLowerCase() === "bitcoincash") {
-          bchData.sighashType = bitcoin.Transaction.SIGHASH_ALL | bitcoin.Transaction.SIGHASH_BITCOINCASHBIP143;
+          bchData.sighashType = bitcoin.Transaction.SIGHASH_ALL | SIGHASH_BITCOINCASHBIP143;
```

Comment:
it looks like this is autodetected by bitcoinjs lib now and should be able to be removed.